app/Platform/RouteServiceProvider.php

```diff
@@ -72,6 +73,7 @@ protected function mapWebRoutes(): void
             // Route::resource('leaderboard', LeaderboardController::class)->only('show');
 
             // Route::get('user/{user}/history', [PlayerHistoryController::class, 'show'])->name('user.history');
+            Route::get('user/{user}/dev-games', DeveloperGameListController::class)->name('user.dev-games');
```

Comment:
Wondering if "dev" should be its own path parameter rather than part of the slug, eg:

`user/{user}/dev/sets`
`user/{user}/dev/games`

This question is coming up because I'm trying to figure out the best path for the Recently Obtained Achievements stuff, eg:

`user/{user}/dev/recent-unlocks`

Reply:
Yup, me too. had something similar in mind:

`/user/{user}/developer` as the main developer profile
`/user/{user}/developer/X` as sub-pages for developer profile related resources and detail pages.
As mentioned, i'd try to keep lingo out of routes and in copy and not shorten "developer" to "dev" to make it sound somewhat official?

Reply:
I think the most fitting route name here would be `developer.games` - substituting `user` with `developer` as an alias in this context 🤔

Reply:
That definitely sounds agreeable to me. Maybe `/user/{user}/developer` takes on individualdevstats.php ?

Reply:
> Maybe /user/{user}/developer takes on individualdevstats.php

Sounds good to me - at least the most applicable atm.
It could house the developer-specific awarded badges for contributions in the future.

Reply:
Throwing `creator` profile in the ring here again, too - just in case we plan on incorporating writers and artists on such pages and keep the "developer" nomenclature specifically for "trigger authors". May as well be just "developers". That's for another discussion - might affect how we want to name routes, though.